schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered boolean,
-    weight_kg DECIMAL(10, 2)
+    weight_kg DECIMAL(10, 2),
+    species VARCHAR(255),
 );
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), kindly add the SQL commands used to add a column `species` of type `string` to your `animals` table.  🤗